x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/action/EnrichCoordinatorStatsAction.java

```diff
@@ -161,6 +184,11 @@ protected NodeResponse nodeOperation(NodeRequest request) {
             DiscoveryNode node = clusterService.localNode();
             return new NodeResponse(node, coordinator.getStats(node.getId()));
         }
+
+        @Override
+        protected String getBwcTransportNodeAction(DiscoveryNode node) {
+            return node.getVersion().before(Version.V_7_7_0) ? BWC_NAME + "[n]" : transportNodeAction;
```

Comment:
This needs to update now that it's slipped.

Reply:
Updated to V_7_9_0